martin/src/cog/model.rs

```diff
@@ -0,0 +1,101 @@
+use std::{fs::File, path::Path};
+
+use tiff::{decoder::Decoder, tags::Tag};
+
+use super::CogError;
+
+// See https://docs.ogc.org/is/19-008r4/19-008r4.html#_coordinate_transformations
+//         ModelPixelScaleTag
+//          ModelTiepointTag
+//  R ------------ OR --------------> M
+// (I,J,K) ModelTransformationTag (X,Y,Z)
```

Comment:
```suggestion
/// See [the ogc docs](https://docs.ogc.org/is/19-008r4/19-008r4.html#_coordinate_transformations) for further details
///
/// The three tags defined below may be used for defining the relationship between R and M, and the relationship may be diagrammed as:
///
/// ```raw 
///        ModelPixelScaleTag
///          ModelTiepointTag
///  R ------------ OR --------------> M
/// (I,J,K) ModelTransformationTag (X,Y,Z)
/// ```
```